### PR TITLE
RIA-3882 Admin officer journey chooses fee update not required after additional fee required

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3882](https://tools.hmcts.net/jira/browse/RIA-3882)

### Change description ###
Admin officer journey chooses fee update not required after additional fee required

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/features/manage-fee-update.feature [hunk 1]
```diff
@@ -903,3 +903,155 @@ Feature: Manage fee update
...
+    And I click the `Continue` button
+
+    Then I am on the `Check your answers` page
+    And I should see `Decision type changed` in the `Reason for fee update` field
+    And I should see `£10.00` in the `New fee amount (£)` field
+    And I should see `Fee update recorded` in the `Fee update status` field
+    And I should see the text `Additional fee requested`
+    And I should see the text `Fee update not required`
+    And I click the `Submit` button
+
```
Comment (divisathyan) [inline e2e/features/manage-fee-update.feature:970]: Missing confirmation page check

File: e2e/features/manage-fee-update.feature [hunk 2]
```diff
@@ -903,3 +903,155 @@ Feature: Manage fee update
...
+    Then I save my initial `EA` appeal type with `no remission` and `with` hearing fee
+    And I pay for and submit my appeal
+
+    When I switch to be a `Admin Officer`
+    Then I select the `Manage a fee update` Next step
+    And I should see the `Manage a fee update` page
+    And I should see the text `Select the reason the fee has been updated.`
+    And I select `Decision type changed` for the `Reason for fee update` field
+    And I type `10` for the `New fee amount (£)` field
```
Comment (divisathyan) [inline e2e/features/manage-fee-update.feature:1010]: Remove the (£) for New amount

File: e2e/features/manage-fee-update.feature [hunk 3]
```diff
@@ -903,3 +903,155 @@ Feature: Manage fee update
...
+    And I click the `Continue` button
+
+    Then I should see the text `You need to record the status of the fee update. To do this select the task you are about to complete. Do not uncheck any previously selected tasks.`
+    And I click the first `Fee update recorded` label
+    And I click the `Continue` button
+
+    Then I am on the `Check your answers` page
+    And I should see `Decision type changed` in the `Reason for fee update` field
+    And I should see `£10.00` in the `New fee amount (£)` field
```
Comment (divisathyan) [inline e2e/features/manage-fee-update.feature:1019]: same as above

**Review by divisathyan — CHANGES_REQUESTED** [review]
Please see comment

File: e2e/features/manage-fee-update.feature (hunk 2 above)
Comment (tarun-palisetty) [reply e2e/features/manage-fee-update.feature:1010]: Removed the £ sign

File: e2e/features/manage-fee-update.feature (hunk 3 above)
Comment (tarun-palisetty) [reply e2e/features/manage-fee-update.feature:1019]: removed

File: e2e/features/manage-fee-update.feature (hunk 1 above)
Comment (tarun-palisetty) [reply e2e/features/manage-fee-update.feature:970]: Added the check

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by divisathyan — APPROVED** [review]

**Pull request merged**